File: diploma-rest-api/src/main/java/org/arhor/diploma/dto/AccountDTO.java
package org.arhor.diploma.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public final class AccountDTO {
  private Long id;
  private String username;
  private String password;
  private String email;
  private String firstName;
  private String lastName;
  private String role;
  private LocalDateTime created;
  private LocalDateTime updated;
  private boolean deleted;
}

File: diploma-server/src/main/kotlin/org/arhor/diploma/service/dto/AccountDTO.kt
package org.arhor.diploma.service.dto

import org.arhor.diploma.commons.Identifiable

data class AccountDTO(
    override var id: Long? = null,
    var username: String? = null,
    var password: String? = null,
    var email: String? = null,
    var firstName: String? = null,
    var lastName: String? = null
) : Identifiable<Long>
